robottelo/decorators/host.py

```diff
@@ -0,0 +1,120 @@
+import logging
+from functools import wraps
+
+import re
+import six
+import unittest2
+
+from robottelo import ssh
+from robottelo.config import settings
+
+if six.PY3:
+    from functools import lru_cache
+elif six.PY2:
+    # workaround only for this module, its not doing exactly the same as
+    # original from Python 3, but it fits this specific case
+    def lru_cache(maxsize):
+        def decorator(func):
+            cache = []
+
+            @wraps(func)
+            def wrapper():
+                if not cache:
+                    cache.append(func())
+                return cache[0]
+
+            return wrapper
+
+        return decorator
+
+_LOGGER = logging.getLogger(__name__)
```

Comment:
I Copy this from here: https://github.com/SatelliteQE/robottelo/blob/master/robottelo/decorators.py#L16. But once my version is used only inside the module, I've made it protected. I've run a "Find Usages" on it and even the on from decorators is used only inside of it. So it is also protectec. From my point of view they could be _LOGGER, because they are never changed and never used outside its own module. What you think @sghai ?